Dockerfile

```
@@ -1,7 +1,12 @@
-FROM alpine:3.17
+FROM ubuntu:22.04
 
-# install packages required to run the tests
-RUN apk add --no-cache jq coreutils
+RUN apt-get update
+RUN apt-get install -y jq coreutils wget zip libfontconfig1
```

This reduces the image and layer size a bit

```suggestion
RUN apt-get update && \
    apt-get install -y jq coreutils wget zip libfontconfig1 && \
    apt-get purge --auto-remove && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

---

Changed, thanks!